<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionFrame</class>
 <widget class="QFrame" name="ConversionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>392</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::WheelFocus</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <attribute name="title">
       <string>Title and cover</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Author string:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLineEdit" name="authorstring">
           <property name="inputMask">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string notr="true">%nf %nm %nl</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="authorTranslit">
           <property name="text">
            <string>Transliteration</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="Label22">
         <property name="text">
          <string>Book title:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="seriastring">
           <property name="inputMask">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string notr="true">(%abbrs %n2) %b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="seriaTranslit">
           <property name="text">
            <string>Transliteration</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="repairCover">
         <property name="text">
          <string>Repair cover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="removePersonal">
         <property name="text">
          <string>Remove 'Personal' tag (AZW3 only)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="createCover">
         <property name="text">
          <string>Create cover if not exists</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="createCaverAlways">
         <property name="text">
          <string>Always create cover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="addCoverLabel">
         <property name="text">
          <string>Add additional label to existing cover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tmplate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Additional label for existing cover:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="coverLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string notr="true">%abbrs - %n2</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Style</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Show footnotes:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="footnotes">
               <item>
                <property name="text">
                 <string>at the end of the book</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>in text</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>after paragraph</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>popup</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>802</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Hyphenate:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="hyphenate">
               <item>
                <property name="text">
                 <string>no</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>soft</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>children</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>802</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Vignette:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QComboBox" name="Vignette">
               <item>
                <property name="text">
                 <string>no</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>image</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>text</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>802</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="break_after_cupture">
             <property name="text">
              <string>Page break between chapters</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="dropcaps">
             <property name="text">
              <string>Insert dropcaps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Content:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QComboBox" name="content_placement">
               <item>
                <property name="text">
                 <string>no content</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>at the beginning of the book</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>at the end of the book</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>802</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>30</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="userCSS">
               <property name="text">
                <string>User CSS</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>802</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="btnDefaultCSS">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="UserCSStext"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>359</width>
            <height>340</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="fontLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="AddFont">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>882</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>644</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="annotation">
         <property name="text">
          <string>Don't add annotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="split_file">
         <property name="text">
          <string>Split into files by chapters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="join_series">
         <property name="text">
          <string>Join a series to single book</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ml_toc">
         <property name="text">
          <string>Use multi-level table of contents</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_lavel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Maximum caption level:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MAXcaptionLevel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
